### PR TITLE
Fix #3384: Arranged Help Activity string resources

### LINKED ISSUE #3384: Arrange string resources based on screens
**Describe the bug**

Currently string resources are not organised correctly and are scattered in `strings.xml` file.

This should be fixed by following these points:
1. Make sure that each string is getting used in one specific view only. For example `Home` is part of multiple views like `Navigation Drawer`, `Home-Toolbar`. So for this we should have separate strings for both.
2. All strings related to one activity/fragment should be kept together by mentioning the activity name on top of the strings in a comment. Example: `<!-- ProfileResetPinActivity -->`
3. All activities/fragment sections should be in correct alphabetical order based on activity/fragment name. (The actual strings name are not required to be in alphabetical order but the comments should be in alphabetical order). Example:
```
<!-- HomeActivity -->
<string name="random_string_related_to_home">ABC</string>
<!-- ProfileResetPinActivity -->
<string name="random_string_related_to_profile_resent_pin">ABC</string>
<string name="random_string_related_to_profile_resent_pin_2">XYZ</string>
```

Arrange strings file as per below criteria:

**Screen/Toolbar Titles**
Used in manifest label, toolbar title and activity label test.
Pattern: `activity_name_activity_title`

Example: `administrator_controls_activty_title`

**App Content Strings**
Pattern: `activity_name_view_component_type`

`administrator_controls_activity_profile_management_label`
`administrator_controls_activity_account_actions_label`
`administrator_controls_activity_logout_item_text`
`administrator_controls_activity_app_version_item`

**Shared Strings**
Pattern: `view_component_description`

reading_text_size_item_text_small
reading_text_size_item_medium

**Test Strings**
Pattern: `test_file_name_view_component_type`

Dialogs
Pattern: `dialog_name_view_component_type`

Grouping Pattern: 

  `<!-- AdministratorControlActivity -->`
….
….
  `<!-- GeneralAvailabilityUpgradeNoticeDialogFragment -->`
…..
   `<!-- ProgressDatabaseFullDialog -->`

### PR DESCRIPTION
**Pull request by TheSwarnim**
## Explanation

Fixes #3384
In this PR, I've arranged HelpAcctivity string resources in the strings.xml file. I've also arranged FAQ string resources in a subheading of HelpActivity.
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -449,4 +446,10 @@
   <string name="correct_submitted_answer_with_append">Correct submitted answer: %s</string>
   <string name="incorrect_submitted_answer">Incorrect submitted answer</string>
   <string name="incorrect_submitted_answer_with_append">Incorrect submitted answer: %s</string>
+
+<!--  Help Activity-->
+  <string name="menu_help">Help</string>
+  <!--  FAQ (Help Activity)-->
```
Comment (anandwana001) [inline app/src/main/res/values/strings.xml]: We can align everything with same indentation, though FAQ comes under Help, but keeping same indentation is kind of standard format we follow.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -5,7 +5,6 @@
   <string name="menu_home">Home</string>
   <string name="menu_options">Options</string>
   <string name="menu_my_downloads">My Downloads</string>
-  <string name="menu_help">Help</string>
   <string name="exploration_activity_title">Exploration Player</string>
   <string name="help_activity_title">Help</string>
```
Comment (anandwana001) [inline app/src/main/res/values/strings.xml:9]: Could you check if help activity is using this or not?

Also, are there any other items which we need to put under help activity.

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (TheSwarnim) [reply app/src/main/res/values/strings.xml:9]: Yes, I've checked it and I've also searched this and found no use in the help activity.
Also, I've searched all items, and only the above all I've found that were used in help activity

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (TheSwarnim) [reply app/src/main/res/values/strings.xml]: Okay, sir, I'll recommit by changing the file.

**Comment by TheSwarnim** [comment]
@anandwana001 sir, please review it again. I've resolved the conversaton

**Review by anandwana001 — APPROVED** [review]
LGTM,

nit suggestion for future PR, the conversation must be resolved only by the PR reviewer.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -449,4 +446,9 @@
   <string name="correct_submitted_answer_with_append">Correct submitted answer: %s</string>
   <string name="incorrect_submitted_answer">Incorrect submitted answer</string>
   <string name="incorrect_submitted_answer_with_append">Incorrect submitted answer: %s</string>
+
+<!--  Help Activity-->
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Nit
1. There should be space at the start.
2. And there should be 1 space before and after text.

`  <!-- Help Activity -->`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@TheSwarnim PTAL

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (TheSwarnim) [reply app/src/main/res/values/strings.xml]: Thanks sir, I've done some changes in file, please review it again

Comment (rt4914) [reply app/src/main/res/values/strings.xml]: @TheSwarnim It does not match with the above comment. Can you please take a look.

Comment (rt4914) [reply app/src/main/res/values/strings.xml]: Also, update your branch by merging with latest develop.

**Comment by anandwana001** [comment]
@TheSwarnim any update on this?

**Comment by rishidyno** [comment]
@anandwana001 I looked into help_activity.xml files no instances of string resources are being used in those files could you verify me if I am wrong.
Also, if @TheSwarnim is not working on this issue could it be assigned to me @anandwana001.

**Comment by anandwana001** [comment]
@rishidyno The issue here is to arrange the resources which are used in the Help Screen (HelpActivity.kt, it's other related file) in `strings.xml` file.

Closing this PR and assigning to you @rishidyno

**Pull request closed without merging**